tests/test_io_elementwise.py

```diff
@@ -206,14 +206,6 @@ def create_sparse_store(
     ],
 )
 def test_io_spec(store, value, encoding_type):
-    # zarr v3 can't write recarray
```

Comment:
oof we should have caught all that in the recarray-enabling PR.

how did you check if you caught all these instances now?